packages/shared/components/popups/CreateAccountPopup.svelte

```diff
@@ -32,6 +34,31 @@
         }
     }
 
+    function create() {
+        tryCreateAdditionalAccount(trimmedAccountAlias, color.toString())
+            .then(() => closePopup)
+            .catch((error) => {
+                isBusy = false
+                if (error) {
+                    console.error(error?.error || error)
+                    if ($isLedgerProfile) {
+                        displayNotificationForLedgerProfile('error', true, false, false, false, error)
+                    } else {
+                        showAppNotification({
+                            type: 'error',
+                            message: localize(error?.error || error),
+                        })
+                    }
+                } else {
+                    closePopup()
+                }
+            })
```

Comment:
Can we convert this to not use callbacks? Also I think `isBusy` is not set to false upon success.

Reply:
This is updated in feat/manage-account-stardust